packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Remove peer dependency `@babel/runtime@^7.23.9`
```

Comment:
Hmm. I came across that earlier, but since it's from 2019, I didn't look to see if this was an intentional choice or it was changed in recent versions of Yarn. But from the tone of the post it does certainly sound intentional.

If peer dependencies aren't inherited does this mean that `@metamask/providers` and `webextension-polyfill` should actually be dev and peer dependencies instead of just "regular" dependencies?

Reply:
Yes, exactly.

To clarify, the post suggests that _regardless_ whether the peer dependencies are transitive/inherited or not, we'd potentially run into broken situations due to hoisting unless we carry the peer dependency through in the intermediary package.